### PR TITLE
Add docs for pipectl event register --contexts on the event watcher usage page

### PR DESCRIPTION
**Pull request by ffjlabo**
**What this PR does**:

as title

**Why we need it**:

**Which issue(s) this PR fixes**:

Part of https://github.com/pipe-cd/pipecd/issues/5028
Related PR #5295 

**Does this PR introduce a user-facing change?**:

- **How are users affected by this change**:
- **Is this breaking change**:
- **How to migrate (if breaking change)**:

### CODE REVIEW

File: docs/content/en/docs-dev/user-guide/command-line-tool.md [hunk 1]
```diff
@@ -328,6 +328,8 @@ pipectl event register \
     --data=gcr.io/pipecd/example:v0.1.0
 ```
 
+See more about [usage for Event Watcher](./event-watcher.md)
```
Comment (t-kikuc) [inline docs/content/en/docs-dev/user-guide/command-line-tool.md:331]: [nit]
```suggestion
See more on [usage of Event Watcher](./event-watcher.md).
```

File: docs/content/en/docs-dev/user-guide/event-watcher.md [hunk 2]
```diff
@@ -165,6 +165,38 @@ pipectl event register \
...
+    --address=CONTROL_PLANE_API_ADDRESS \
+    --api-key=API_KEY \
+    --name=sample \
+    --data=gcr.io/pipecd/helloworld:v0.48.0 \
+    --contexts Source-Commit-Hash=xxxxxxx, Source-Commit-URL=https://github.com/pipe-cd/pipecd/commit/xxxxxxx
+```
+
+```bash
+# For manifest repository
+% git show
```
Comment (t-kikuc) [inline docs/content/en/docs-dev/user-guide/event-watcher.md:187]: [nit] you mean this?
```suggestion
# In manifest repository
$ git show                                                                        
```

**Review by t-kikuc — COMMENTED** [review]
Thank you, I commented 2 nits

File: docs/content/en/docs-dev/user-guide/command-line-tool.md (hunk 1 above)
Comment (ffjlabo) [reply docs/content/en/docs-dev/user-guide/command-line-tool.md]: Fixed at [09c9cc2](https://github.com/pipe-cd/pipecd/pull/5299/commits/09c9cc20498fa6a542080cd8eb6aa12c81b71bd5)

File: docs/content/en/docs-dev/user-guide/event-watcher.md (hunk 2 above)
Comment (ffjlabo) [reply docs/content/en/docs-dev/user-guide/event-watcher.md]: Fixed [09c9cc2](https://github.com/pipe-cd/pipecd/pull/5299/commits/09c9cc20498fa6a542080cd8eb6aa12c81b71bd5)
Thanks, '%' is my terminal prompt. changed to the '$'

**Review by t-kikuc — APPROVED** [review]

File: docs/content/en/docs-dev/user-guide/event-watcher.md [hunk 3]
```diff
@@ -165,6 +165,38 @@ pipectl event register \
...
+
+For example, you can attach the source code commit link to the manifest repository.
+
+```bash
+pipectl event register \
+    --address=CONTROL_PLANE_API_ADDRESS \
+    --api-key=API_KEY \
+    --name=sample \
+    --data=gcr.io/pipecd/helloworld:v0.48.0 \
+    --contexts Source-Commit-Hash=xxxxxxx, Source-Commit-URL=https://github.com/pipe-cd/pipecd/commit/xxxxxxx
```
Comment (Warashi) [inline docs/content/en/docs-dev/user-guide/event-watcher.md:182]: We have to surround the argument with quotes, or when we don't surround it we cannot put a space in the argument.
```suggestion
    --contexts Source-Commit-Hash=xxxxxxx,Source-Commit-URL=https://github.com/pipe-cd/pipecd/commit/xxxxxxx
```

Comment (ffjlabo) [reply docs/content/en/docs-dev/user-guide/event-watcher.md]: Thank you, fixed it at [4a2334a](https://github.com/pipe-cd/pipecd/pull/5299/commits/4a2334a3c96d31c9bcfcbfae02eb6873dd923cdc)

**Review by Warashi — APPROVED** [review]
Thank you

**Review by t-kikuc — APPROVED** [review]

**Pull request merged**